source/query-bar.txt

```diff
@@ -268,24 +276,23 @@ Past Queries View
 -----------------
 
 Compass automatically stores up to 20 most recent queries for each
-collection. To view these queries, click on the ellipsis
+collection. To view these queries, click the ellipsis
 :guilabel:`(...)` button in the query bar, then click
 :guilabel:`Toggle Query History`.
 
 .. figure:: /images/compass/query-history-view.png
 
 From the :guilabel:`Past Queries` view, you can view
-:ref:`recent queries <recent-queries>` as well as
-the queries saved as :ref:`favorites <favorite-queries>` for
-the current collection.
+:ref:`recent queries <recent-queries>` and queries saved as
+:ref:`favorites <favorite-queries>` for the current collection.
 
 .. _recent-queries:
 
 Recent Queries
 ~~~~~~~~~~~~~~
 
-You can view up to 20 most recent queries for that collection. Click on
-a listed query to populate the query bar with that query.
+You can view up to the 20 most recent queries for that collection.
```

Comment:
You can view up to 20 of the most recent queries...

Reply:
Much better. Ty!